rest_framework_json_api/views.py

```diff
@@ -154,7 +154,7 @@ def get_related_serializer(self, instance, **kwargs):
         return serializer_class(instance, **kwargs)
 
     def get_related_serializer_class(self):
-        parent_serializer_class = super(RelatedMixin, self).get_serializer_class()
```

Comment:
Explanation 1:
This is the actual issue this PR is trying to solve.
Getting the `super()` serializer class will call vanilla DRF, which raises the exception:
> 'EntryViewSet' should either include a `serializer_class` attribute, or override the `get_serializer_class()` method.